jenkins-config/clouds/openstack/Primary/centos7-docker-4c-4g.cfg

```diff
@@ -0,0 +1,2 @@
+IMAGE_NAME=ZZCI - CentOS 7 - docker - x86_64 - 20190417-182736.971
+HARDWARE_ID=v1-standard-4
```

Comment:
You should go ahead and change this to v2-standard-4

Reply:
@MightyNerdEric - Updated the PR with requested change